index.d.ts

```diff
@@ -0,0 +1,59 @@
+export interface Options {
+	/**
+	 * Default: [Only in development](https://github.com/sindresorhus/electron-is-dev)
+	 */
+	enabled?: boolean;
+
+	/**
+	 * Show DevTools on each created `BrowserWindow`.
+	 *
+	 * @default true
+	 */
+	showDevTools?: boolean;
+
+	/**
+	 * The dock state to open DevTools in.
+	 *
+	 * @default 'undocked'
+	 */
+	devToolsMode?: 'undocked' | 'right' | 'bottom' | 'previous' | 'detach';
+}
+
+/**
+ * Install keyboard shortcuts and optionally activate DevTools on each created `BrowserWindow`.
+ *
+ * @example
+ *
+ * import {app, BrowserWindow} from 'electron';
+ * import electronDebug from 'electron-debug';
+ *
+ * electronDebug();
+ *
+ * let win;
+ * (async () => {
+ * 	await app.whenReady();
+ * 	win = new BrowserWindow();
+ * });
+ */
+export default function electronDebug(options?: Options): void;
+
+/**
+ * Reload the specified `BrowserWindow` instance or the focused one.
+ *
+ * @param window - Default: BrowserWindow.getFocusedWindow()
+ */
+export function refresh(window?: Electron.BrowserWindow): void;
```

Comment:
Couldn't you use a default parameter here?

```js
export function refresh(window?: Electron.BrowserWindow = BrowserWindow.getFocusedWindow()): void;
```

Then line 43 would be moot.

Reply:
Hm, the default value doesn't seem to show up in the inline docs preview when using this syntax.

Reply:
But it’s valid syntax, right? If so, I think we should still use it and open an issue on VSCode about it.